tests/integration/ambient/main_test.go

```diff
@@ -420,3 +423,15 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 
 	return nil
 }
+
+func SetupSuite(ctx resource.Context) (err error) {
```

Comment:
can we move these steps inside testRegistrySetup() and remove this additional function?

Reply:
done